tests/helpers/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""Test helpers for snitun tests."""
```

Comment:
It would be good to extend the benchmarks with specific metrics benchmarks also.

Reply:
The main impact is going to be counting them all in the loop. Most metric reporters will fire them via UDP to a collector agent to avoid blocking. We could build a big dict and iterate on it to emulate this impact for a benchmark, but realistically whether the performance impact is going to be a degree of how many connections are on each node (iteration loop size) and how fast the machine is. I expect this will need to be adjusted for production based on those factors, thus the configurable interval.